README.md

```diff
@@ -95,37 +95,28 @@ doc](https://docs.google.com/document/d/1i1E4-_y4uJ083lCutKGDhkpi3n4_e774SBLi9hP
 For edit access, get in touch on
 [Slack](https://cloud-native.slack.com/archives/C01N3AT62SJ).
 
-Approvers
-([@open-telemetry/cpp-approvers](https://github.com/orgs/open-telemetry/teams/cpp-approvers)):
-
-* [Ehsan Saei](https://github.com/esigo)
-* [Johannes Tax](https://github.com/pyohannes), Microsoft
-* [Josh Suereth](https://github.com/jsuereth), Google
-* [Max Golovanov](https://github.com/maxgolov), Microsoft
-* [Ryan Burn](https://github.com/rnburn), Lightstep
-
-*Find more about the approver role in [community
-repository](https://github.com/open-telemetry/community/blob/main/community-membership.md#approver).*
-
-Maintainers
+[Maintainers](https://github.com/open-telemetry/community/blob/main/community-membership.md#maintainer)
 ([@open-telemetry/cpp-maintainers](https://github.com/orgs/open-telemetry/teams/cpp-maintainers)):
 
-* [Emil Mikulic](https://github.com/g-easy), Google
 * [Lalit Kumar Bhasin](https://github.com/lalitb), Microsoft
 * [Reiley Yang](https://github.com/reyang), Microsoft
 * [Tom Tan](https://github.com/ThomsonTan), Microsoft
 
-*Find more about the maintainer role in [community
-repository](https://github.com/open-telemetry/community/blob/main/community-membership.md#maintainer).*
+[Approvers](https://github.com/open-telemetry/community/blob/main/community-membership.md#approver)
```

Comment:
Should we be updating the approver's [team ](https://github.com/orgs/open-telemetry/teams/cpp-approvers/members) accordingly as suggested [here](https://github.com/open-telemetry/community/blob/main/community-membership.md#emeritus-maintainerapprovertriager).
Same for maintainers.

Reply:
` GitHub usernames listed under the emeritus section should be removed from the approvers list, and will no longer have any official responsibilities with the project. ` - Perhaps it's only removing from the README.md list, not the team. Ignore my comment in that case :)